packages/authereum-connector/src/index.ts

```diff
@@ -47,4 +47,9 @@ export class AuthereumConnector extends AbstractConnector {
   }
 
   public deactivate() {}
+
+  public async close() {
```

Comment:
I don't know why this is showing up in this PR because it was already merged into v6